components/textfield/index.css

```diff
@@ -617,12 +617,6 @@ governing permissions and limitations under the License.
 		);
 		grid-row: 1;
 		grid-column: 1 / span 1;
-		padding-inline-start: calc(
```

Comment:
@jawinn @pfulton Is there a risk at all that this could break a downstream override?  Should we keep the property with the mod + calc and just remove the fallback?